packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -802,6 +802,39 @@ class ListTile extends StatelessWidget {
       ),
     );
   }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<Widget>('leading', leading, defaultValue: null));
+    properties.add(DiagnosticsProperty<Widget>('title', title, defaultValue: null));
+    properties.add(DiagnosticsProperty<Widget>('subtitle', subtitle, defaultValue: null));
+    properties.add(DiagnosticsProperty<Widget>('trailing', trailing, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('isThreeLine', isThreeLine, defaultValue: false));
```

Comment:
The bool properties should be `FlagProperty` instead of `DiagnosticsProperty<bool>`.

Then you can add an `ifTrue` set to "THREE_LINE" or something.